src/core/qgsogrutils.cpp

```diff
@@ -98,6 +98,89 @@ void gdal::GDALWarpOptionsDeleter::operator()( GDALWarpOptions *options )
   GDALDestroyWarpOptions( options );
 }
 
+QVariant QgsOgrUtils::OGRFieldtoVariant( const OGRField *value, OGRFieldType type )
+{
+  if ( !value )
```

Comment:
```suggestion
  if ( !value || OGR_RawField_IsUnset(value) )
```

Reply:
Should I also check OGR_RawField_IsNull here?

Reply:
This will not normally happen for range domain fields, but for generality that'd be better indeed.

Reply:
Ok, done